application/app/views/layouts/nav/_navigation.html.erb

```diff
@@ -49,23 +49,28 @@
              href="#repo-resolver-bar"
              role="button"
              aria-expanded="false"
+             data-controller="focus-on-event"
+             data-action="keydown.enter->focus-on-event#focusInput click->focus-on-event#focusInput"
+             data-focus-on-event-event-value="shown.bs.collapse"
+             data-focus-on-event-element-id-value="input_repo_url"
```

Comment:
The idea is to use the event or the data-action, but not both.
For components with events, like BS collapse, it can be used to ensure the collapse action has completed.

for other items, the data-action will be used.